core/constants.go

```diff
@@ -476,8 +476,8 @@ const (
 	ConsensusOrder
 	// NetworkShardingOrder defines the order in which the network sharding subsystem is notified of a start of epoch event
 	NetworkShardingOrder
-	// OutportOrder defines the order in which Outport is notified of a start of epoch event
-	OutportOrder
+	// IndexerOrder defines the order in which indexer is notified of a start of epoch event
+	IndexerOrder
```

Comment:
OutportOrder ? We should decide if we should keep outport or indexer naming

Reply:
I will keep IndexerOrder --> If I change in Outport order we need a new release on the elastic-indexer repository.

Reply:
ok